filedistribution/src/vespa/filedistribution/distributor/filedownloader.cpp

```diff
@@ -285,7 +285,7 @@ FileDownloader::pathToCompletedFile(const std::string& fileReference) const {
 
 boost::optional<FileDownloader::ResumeDataBuffer>
 FileDownloader::getResumeData(const std::string& fileReference) {
-    LOG(debug, ("Reading resume data for " + fileReference).c_str());
+    LOG(debug, "%s", ("Reading resume data for " + fileReference).c_str());
```

Comment:
For this and similar log sites in this file, could move string concat into the log format string itself, i.e. `LOG(debug, "Reading resume data for %s", fileReference.c_str());` to avoid the temp string alloc, but this isn't exactly a hot inner loop...

Reply:
Fixed too